scripts/library_report

```diff
@@ -72,7 +70,13 @@ try:
   print("randomENG","\n",randomENG)
 except ImportError:
   print("randomENG not found\n")
-
+print()
+amsc_report = library_handler.checkSingleLibrary('AMSC', useImportCheck=True)
+for module, found, message, version in [('AMSC',amsc_report[0], amsc_report[1], "")]:
```

Comment:
why do you need a for loop here? It seems the version is an empty string, why do we need it here?

Reply:
Good point, I just copied the previous loop, but I will change that.